framework/test_framework.py

```diff
@@ -222,5 +249,7 @@ def deploy_test(platform):
                cons.RESET_COLOR)
         sys.exit(-1)
 
+    move_results_to_output()

```

Comment:
Does the remaining framework (i.e., qemu launcher) consider the new directory of the binaries?

Reply:
Good point, thank you! Indeed, the platform deployment call takes into account the location of these artifacts. You can review it here:
https://github.com/bao-project/bao-tests/blob/feat/build-artifacts-management/framework/test_framework.py#L140-L145

The function that search in the output folder can be checked here:
https://github.com/bao-project/bao-tests/blob/feat/build-artifacts-management/framework/test_framework.py#L107-L126